packages/insomnia-app/app/common/misc.js

```diff
@@ -359,6 +360,10 @@ export async function waitForStreamToFinish(s: Readable | Writable): Promise<voi
   });
 }
 
+export function getDesignerDataDir(): string {
+  return envPaths('Insomnia Designer', { suffix: '' }).data;
```

Comment:
This won't work correctly on Windows, as `env-paths` returns `%LOCALAPPDATA%/Insomnia Designer/Data` instead of the correct `%APPDATA%/Insomnia Designer%`.

I think it'd be better to use the Electron APIs:

```suggestion
  const { app } = electron.remote || electron;
  return pathJoin(app.getPath('appData'), 'Insomnia Designer');
```

Reply:
I have updated this, but cannot verify on Windows until later this evening. Nice catch though, I've removed envPaths as an app dependency as well.